Dangerfile

```diff
@@ -0,0 +1,22 @@
+# PR too big
+MAX_PR_SIZE = 250
+warn("PR is exceeds `#{MAX_PR_SIZE}` LoC. Consider breaking up into multiple smaller ones.") if git.lines_of_code > MAX_PR_SIZE
```

Comment:
```suggestion
if git.lines_of_code > MAX_PR_SIZE
  warn("Pull request exceeds `#{MAX_PR_SIZE}` lines of code. Consider breaking up into multiple smaller pull requests.")
end
```

Nitpicky but a bit easier to parse the `if` block imo

Reply:
Agree.